app/controller/button/DigitizeButtonController.js

```diff
@@ -341,11 +349,17 @@ Ext.define('CpsiMapview.controller.button.DigitizeButtonController', {
             }
         }
 
+        me.setDrawInteraction(me.drawLayer);
+        me.setModifyInteraction(me.drawLayer);
+        me.setPointerInteraction();
```

Comment:
is there intentionally no argument?

Reply:
the pointer interaction does not have a layer filter, but interacts with the `drawLayer` directly so it is not needed to pass it here.

Reply:
alright